doc/website/sidebars.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
                 {type: 'doc', id: 'versioned/diff', label: 'Migration Authoring'},
                 {type: 'doc', id: 'versioned/lint', label: 'Migration Linting'},
                 {type: 'doc', id: 'versioned/new', label: 'Manual Migrations'},
+                {type: 'doc', id: 'versioned/execution', label: 'Migration Execution'},
```

Comment:
Remove it from now until we release it on Monday. 
```suggestion
                {type: 'doc', id: 'versioned/apply', label: 'Migration Execution'},
```